src/test/scala/tech/cryptonomic/conseil/tezos/InMemoryDatabase.scala

```diff
@@ -39,15 +38,6 @@ trait InMemoryDatabase extends BeforeAndAfterAll with BeforeAndAfterEach {
        |  }
     """.stripMargin
 
-  /* turns off anti-corruption guarantees settings that will improve performance on testing
-   * override to change or add test-specific settings
-   */
-  protected val pgInitParams = List("--nosync", "--lc-collate=C")
-  /* turns off anti-corruption guarantees settings that will improve performance on testing
-   * override to change or add test-specific settings
-   */
-  protected val pgConfigs = List("-c", "full_page_writes=off")
-
```

Comment:
I guess we remove these because they were only available to the embedded pg library?